### ISSUE TITLE
MS File Reader installation/version

### ISSUE THREAD

**Issue by SSPuliasis** [vdemichev/DiaNN#570]
Installing MSFileReader gives this error:

'Could not launch the Microsoft Visual C++ 2008 x64 Redistributables Installer'

Installation of the specified version of MS File Reader requires a specific version of Microsoft Visual C++ 2008, which is not on my windows 10 computer (versions 2010, 2013, 2015 are). Trying to install the 2008 version from Microsoft is no longer possible (https://www.microsoft.com/en-us/download/404Error.aspx)

Does DIA-NN support any more recent versions of MS File Reader, or will it in the future? If anyone else has encountered this problem and found a way around it, please let me know

Thanks

**Comment by vdemichev** [comment]
No, only works with the specific MSFileReader version, and it is known to work fine on modern systems too. Do you install it from an account that has administrator rights?

**Comment by SSPuliasis** [comment]
Thanks for the quick reply

Yes, I'm installing it from an account with admin rights but that doesn't seem to be the issue. 

When installing Thermo MS File Reader 3.0, I get an error saying 'Could not launch the Microsoft Visual C++ 2008 x64 Redistributables Package'. As mentioned I have all the versions of this redistributables package from 2010 onwards and could not find a reliable 2008 version online so I contacted Microsoft. 

According to Microsoft, reverting back to older Microsoft framework would allow for the installation of MS File Reader 3.0, but would likely disrupt any software I have that is dependent on the newer framework, so they advised against it. I'm waiting to hear back from Thermo Fisher about providing an updated installer, but am not sure if they'll do it for MS File Reader 3.0 when newer versions are available.

I would really like to use DIA-NN for my DIA analyses, and am mentioning this here in case any other users with newer PCs encounter the same issue. Hopefully it can get resolved
